docs/sources/explore/trace-integration.md

```diff
@@ -8,167 +7,159 @@ labels:
+You can expand any span in a trace and view the details, including the span and resource attributes.
 
-Shows list of spans within the trace. Each span row consists of these components:
+For more information about spans and traces, refer to [Introduction to tracing](https://grafana.com/docs/tempo/latest/introduction/) in the Tempo documentation.
 
-- Expand children button: Expands or collapses all the children spans of selected span.
-- Service name: Name of the service logged the span.
-- Operation name: Name of the operation that this span represents.
-- Span duration bar: Visual representation of the operation duration within the trace.
+Span details include:
 
-Clicking anywhere on the span row shows span details.
+- **Span attributes** - Key/value pairs that provides context for spans. For example, if the span deals with calling another service via HTTP, an attribute could include the HTTP URL (maybe as the span attribute key `http.url`) and the HTTP status code returned (as the span attribute `http.status_code`).
 
-### Span details
+- **Resource attributes** - Key/value pairs that describe the context of how the span was collected.
 
-{{< figure src="/media/docs/tempo/screenshot-grafana-trace-view-span-details.png" class="docs-image--no-shadow" max-width= "900px"  caption="Screenshot of the trace view span details" >}}
+Refer to [Span and resource attributes](/docs/tempo/<TEMPO_VERSION>/operations/best-practices/#span-and-resource-attributes) for more detail.
 
-- Operation name.
+<!-- - Operation name.
 - Span metadata.
 - Tags: Any tags associated with this span.
 - Process metadata: Metadata about the process that logged this span.
-- Logs: List of logs logged by this span and associated key values. In case of Zipkin logs section shows Zipkin annotations.
+- Logs: List of logs logged by this span and associated key values. In case of Zipkin logs section shows Zipkin annotations. -->
 
-### Trace to logs
+{{< figure src="/media/docs/tempo/screenshot-grafana-trace-view-span-details.png" class="docs-image--no-shadow" max-width= "900px"  caption="Trace view span details" >}}
 
-You can navigate from a span in a trace view directly to logs relevant for that span. This feature is available for Tempo, Jaeger, and Zipkin data sources. Refer to their [relevant documentation](/docs/grafana/latest/datasources/tempo/#trace-to-logs) for configuration instructions.
+### Span filters
 
-{{< figure src="/media/docs/tempo/screenshot-grafana-trace-view-trace-to-logs.png" class="docs-image--no-shadow" max-width= "900px" caption="Screenshot of the trace view in Explore with icon next to the spans" >}}
+Span filters allow you to refine the spans displayed in the trace timeline viewer. The more filters you add, the more specific the filtered spans become. Click on a trace to access Span filters.
```

Comment:
Again, one sentence per line. It's not always followed, but it can make editing text easier. 
```suggestion
Span filters allow you to refine the spans displayed in the trace timeline viewer.
The more filters you add, the more specific the filtered spans become.
Click on a trace to access Span filters.
```